docs/troubleshooting/generating-logs.md

```diff
@@ -9,9 +9,9 @@ Both logger options are configured from the `Settings` page.
 > [!NOTE]
 > Application logs are of limited utility and are only useful for diagnosing **errors in the UI itself**, versus lower-level Z-Wave network operations that are reflected in the driver logs.
 
-ZWavejs2Mqtt logger can be configured in `General` section, enable `Log enabled` switch and `Log To File` and set `Log Level` to `Silly`
+Z-Wave JS UI logger can be configured in `General` section, enable `Log enabled` switch and `Log To File` and set `Log Level` to `Silly`
 
-> Log file name: `zwavejs2mqtt.log`
+> Log file name: `zwave-js-ui.log`
```

Comment:
We should change this to be very obviously the UI log. Users are already confusing the `zwavejs2mqtt.log` file with `zwave-js.log`. This is even closer now.

Reply:
You mean in the breaking changes?

Reply:
Changed to `z-ui.log`